src/backend/app/utils.py

```diff
@@ -527,3 +528,40 @@ def geojson_to_kml(geojson_data: dict) -> str:
     kml_output.append("</kml>")
 
     return "\n".join(kml_output)
+
+
+async def send_project_approval_email_to_regulator(
+    emails: list, project_id: str, creator: str, project_name: str
+):
+    for email in emails:
+        encoded_email = base64.urlsafe_b64encode(email.encode()).decode()
+        project_link = (
+            f"{settings.FRONTEND_URL}/projects/{project_id}/?token={encoded_email}"
+        )
+
+        body = f"""
+        Hello,
+
+        A project has been created by {creator}, and you have been designated as the regulator for this project: {project_name}.
+        Approval Link: {project_link}
+
+        Note: Drone operator will be able to access and view the project only after your approval.
+        """
+
+        message = MIMEText(body, "plain")
+        message["From"] = formataddr(
```

Comment:
If possible, can we avoid the for loop for sending emails? Instead of writing the email-sending logic from scratch, we already have a `send_notification_email` function in the `utils` module that can be used.